### PR TITLE
Add notification's emails HTML alternative + subject translation

### PR DESCRIPTION
**Pull request by DylannCordel**
Related to #167

* Emails are now sent in text + an HTML alternative (two templates available). Test added.
* Email's subject was outside the "for loop" with user's locale activation and emails was sent with last post author locale. Correction and tests added.

### CODE REVIEW

File: pybb/compat.py [hunk 1]
```diff
@@ -2,7 +2,44 @@
...
+from pybb import defaults
 
+if defaults.PYBB_USE_DJANGO_MAILER:
+    try:
+        from mailer import send_html_mail, send_mail
+    except:
+        pass
+
+try:
+    send_mass_html_mail
```
Comment (Zopieux) [inline pybb/compat.py]: Did you mean `send_html_mail` here? The `mass` one will always trigger the exception as it's not imported/defined anywhere before.

File: pybb/compat.py [hunk 2]
```diff
@@ -2,7 +2,44 @@
...
+except:
```
Comment (Zopieux) [inline pybb/compat.py]: Please use the [narrower `except ImportError`](http://stackoverflow.com/a/14797508/180709).

**Comment by Zopieux** [comment]
Don't forget to wrap text within `{% autoescape off %}` in all the plain-text mail templates. If you don't, the user `Belieber<3` will be rendered as `Belieber&lt;3` in the mail client.

**Comment by DylannCordel** [comment]
Thanks for your feedback @Zopieux
For the `try/except`, I removed it : if user set pybb to use `mailer` but that this one can not be imported, IMHO it's better to _not_ intercept the exception.

**Comment by DylannCordel** [comment]
@GeyseR : could you restart the travis buid please ? It failed becauseof a network bug from travis and/or github. Thx !

**Comment by DylannCordel** [comment]
rebased with master

**Comment by DylannCordel** [comment]
up...

**Pull request merged**